iterreg/sparse/solvers.py

```diff
@@ -167,7 +167,7 @@ def cd_primal_dual(X, y, prox=shrink, max_iter=100, f_store=10, verbose=False):
 
 @njit
 def cd(X, y, alpha, prox=shrink, pen=ell1, max_iter=1_000,
-       f_store=10, w_init=None, verbose=False):
+       f_store=10, w_init=None, ES=False, verbose=False):
```

Comment:
avoid upper case for variables (tolerated for math symbols). 

use either:
`early_stop` 

or even better:
add a sclar tol parameter, equal to 0 by default, and put tol instead if 1e-10 below